src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -255,6 +299,27 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
     this.fire("hass-more-info", { entityId: ev.model.entity.entity_id });
   }
 
+  entityCopyValue(ev) {
+    var action = ev.currentTarget.attributes.action.value;
+    if (action === "copyId") {
+      copy(ev.model.entity.entity_id);
+    } else if (action === "copyState") {
+      copy(ev.model.entity.state);
+    } else if (action === "copyAttributes") {
+      var td = this.shadowRoot.getElementById(
```

Comment:
You can find the `td` based on the target of the event, that saves you from adding an `id` and is easier to lookup. But if you want it to always work, use `this.attributeString(ev.model.entity)`

Reply:
Thanks a lot! I could'nt find how to use this function from within the event...